redisinsight/ui/src/components/consents-settings/ConsentOption/components/ItemDescription.tsx

```diff
@@ -1,26 +1,29 @@
-import { EuiLink } from '@elastic/eui'
 import parse from 'html-react-parser'
 import React from 'react'
+import { Link } from 'uiSrc/components/base/link/Link'
 
 interface ItemDescriptionProps {
-  description: string;
-  withLink: boolean;
+  description: string
+  withLink: boolean
 }
 
-export const ItemDescription = ({ description, withLink }: ItemDescriptionProps) => (
-    <>
-      {description && parse(description)}
-      {withLink && (
-        <>
-          <EuiLink
-            external={false}
-            target="_blank"
-            href="https://redis.io/legal/privacy-policy/?utm_source=redisinsight&utm_medium=app&utm_campaign=telemetry"
-          >
-            Privacy Policy
-          </EuiLink>
-          .
-        </>
-      )}
-    </>
-)
\ No newline at end of file
+export const ItemDescription = ({
+  description,
+  withLink,
+}: ItemDescriptionProps) => (
+  <>
+    {description && parse(description)}
+    {withLink && (
+      <>
+        <Link
```

Comment:
Tip: It's not related anyhow to the visual changes you made, but if you want, you can move this URL to live next to the rest of the `EXTERNAL_LINKS` [here](https://github.com/redis/RedisInsight/blob/main/redisinsight/ui/src/constants/links.ts).

And then, you can use `getUtmExternalLink` to build up the URL with all the necessary query parameters, instead of having them hardcoded.

```suggestion
<Link
    color="primary"
    target="_blank"
    href={getUtmExternalLink(EXTERNAL_LINKS.legalPrivacyPolicy, {
        medium: UTM_MEDIUMS.App,
        campaign: 'telemetry',
    })}}
>
    Privacy Policy
</Link>
```